src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx

```diff
@@ -1166,29 +1165,14 @@ export const SpanBarRectangle = styled('div')<{spanBarHatch: boolean}>`
   ${p => getHatchPattern(p, '#dedae3', '#f4f2f7')}
 `;
 
-const MeasurementMarker = styled('div')<{hovering: boolean}>`
+const MeasurementMarker = styled('div')`
   position: absolute;
-  top: 0;
-  height: ${SPAN_ROW_HEIGHT}px;
-  width: 1px;
+  top: 2px;
+  height: 100%;
   user-select: none;
-  background-color: ${p => p.theme.gray800};
-
-  transition: opacity 125ms ease-in-out;
+  border-left: 2px dashed ${p => p.theme.gray800};
```

Comment:
Can you make this as `border-left` and `border-right` with `1px dashed`? Otherwise, the marker would need to be shifted to the left by `1px`.

Reply:
Ah ok